packages/widget-chat/src/reducers/conversation.js

```diff
@@ -1,21 +1,29 @@
-import {CREATE_CONVERSATION, RECEIVE_CONVERSATION} from '../actions/conversation';
+import {
+  CREATE_CONVERSATION,
+  RECEIVE_CONVERSATION,
+  RECEIVE_MERCURY_ACTIVITY,
+  UPDATE_MERCURY_STATE
+} from '../actions/conversation';
 
 export default function conversation(state = {
```

Comment:
I generally agree as well, but went with the usual nomenclature I've see in reducers. In this specific case, what would you name the function?